build/version/documentdb.go

```diff
@@ -20,8 +20,8 @@ const (
 		"compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit"
 
 	// DocumentDB is a version of DocumentDB this version of FerretDB is compatible with.
-	DocumentDB = "0.102.0 gitref: ferretdb sha:b5ed892 buildId:0"
+	DocumentDB = "0.102.0 gitref: ferretdb sha:b8cd669 buildId:0"
```

Comment:
I updated to the latest sha of ferretdb branch, https://github.com/FerretDB/documentdb/commits/ferretdb/. Is that how we handle sha?

Reply:
Yes, plus `git submodule update --remote` I just did

Reply:
Thanks :pray:

Reply:
Actually, the easiest way is to pull the new image and run `task unit-test`

Reply:
Great!